Added API key setter[DO NOT REVIEW/MERGE]

telescopic opened the pull request:
* Added .pyc file for setting API keys on systems where the api key is not set as an environment variable
* Works for linux systems
* passphrase required to set the API key.

detel commented:
Why do we need to checkin this code? 
https://stackoverflow.com/questions/32110126/should-i-put-pyc-files-under-version-control

Can you explain in detail about this file?

telescopic commented:
> Why do we need to checkin this code?
> https://stackoverflow.com/questions/32110126/should-i-put-pyc-files-under-version-control
> 
> Can you explain in detail about this file?

Thanks for the link. It suggests that .pyc files don't be included and so i guess i'll remove this PR.

I added this so the API key can be directly added by executing this script - this would enable the end-to-end to be executed a bit more easily.
I also added pyc instead of .py since it'll be easier to extract it from the human readable code
Also, the .py filed that was used to compile this requires a passphrase to properly retrieve the API key - this is an extra layer of security added to ensure the API key cannot be easily retrieved.
